src/goto-symex/symex_target.h

```diff
@@ -63,21 +69,46 @@ class symex_targett
     GUARD,
   };
 
-  // read event
+  /// Reading from a shared variable \c ssa_rhs (which will thus become a
+  /// left-hand side as well): we effectively assign the value stored in
+  /// \c ssa_rhs by another thread to \c ssa_rhs in the memory scope of this
+  /// thread.
+  /// \param guard: condition to take this read event
```

Comment:
:pick: This sentence is strange, how about `precondition for this read event` :strawberry:

Reply:
Done.